scripts/irods/test/test_all_rules.py

```diff
@@ -1157,3 +1157,8 @@ def test_rename_to_existing_collection_with_different_name_is_an_error__issue_54
         cmd = ['irule', '-r', rep_name, 'msiRenameLocalZoneCollection("{0}")'.format(existing_collection), 'null', 'ruleExecOut']
         self.admin.assert_icommand(cmd, 'STDERR', ['-809000 CATALOG_ALREADY_HAS_ITEM_BY_THAT_NAME'])
 
+    @unittest.skipUnless(plugin_name == 'irods_rule_engine_plugin-irods_rule_language', 'only applicable for irods_rule_language REP')
+    def test_use_lowercase_select_in_genquery_conditions__issue_4697(self):
+        rep_name = 'irods_rule_engine_plugin-irods_rule_language-instance'
+        cmd = ['irule', '-r', rep_name, '-F', '/var/lib/irods/clients/icommands/test/rules/rulemsiMakeGenQuery.r', "*Coll='/tempZone/home/rods/select/%%'"]
```

Comment:
why two percent trailing signs on the `*Coll` argument?

Reply:
hrm.  that's what is in the rulefile itself...   and was the example that triggered this issue.

I've dropped it to a single `%` and am now using a testfile to match as well.